boards/arm/qemu_cortex_a53/board.cmake

```diff
@@ -5,9 +5,21 @@ set(EMU_PLATFORM qemu)
 set(QEMU_ARCH aarch64)
 
 set(QEMU_CPU_TYPE_${ARCH} cortex-a53)
+
+if(CONFIG_ARMV8_A_NS)
+if (CONFIG_SMP)
+set(QEMU_MACH virt,gic-version=3,virtualization=on)
```

Comment:
I don't get why you need to set `virtualization=on` here and what's the relation with SMP and NS.

When `virtualization` is set to `on` the PSCI conduit is changed from `hvc` to `smc`, so you could simply leave `virtualization=on` out and just change the PSCI conduit from `smc` to `hvc` and everything keeps working.

In theory we should read the conduit directly from the DTS when parsing the `psci` node to set the correct conduit.

If you leave `virtualization` to `off` you are basically offloading the Arm Virtualization Extensions to QEMU and that's something that we actually want.